src/actor-cell.lisp

```diff
@@ -104,12 +104,12 @@ State of the cell can be changed via `setf`ing `*state*` variable."))
 
 (defgeneric stop (actor-cell &optional wait)
   (:documentation "Stops the actor-cells message processing gracefully.
-This is not an immediate stop.  
```

Comment:
Are you on Windows? Wondering where all those whitespace changes are from?

Reply:
No, on Linux, openSUSE Leap 15.5, but my emacs has been configured to delete trailing white spaces when saving.

Reply:
Alright, thanks.